capella2polarion/connectors/polarion_worker.py

```diff
@@ -425,36 +461,104 @@ def compare_and_update_work_items(
             if uuid in self.polarion_data_repo and data.work_item is not None:
                 self.compare_and_update_work_item(data)
 
-    def post_documents(self, documents: list[polarion_api.Document]):
+    def create_documents(
+        self,
+        document_datas: list[data_models.DocumentData],
+        document_project: str | None = None,
+    ):
         """Create new documents."""
```

Comment:
`document_project == None` means the sync project is taken. This needs to be documented:
```suggestion
        """Create new documents.

        Notes
        -----
        If the ``document_project`` is ``None`` the default client is
        taken.
        """
```